A3A/addons/scrt/Stringtable.xml

```diff
@@ -387,6 +387,13 @@
                 <Korean>호라이즌 제도 방위군(HIDF).\n호라이즌 제도 공화국의 군대이며, 많은 여분의 NATO 장비를 사용합니다.</Korean>
                 <French>Force de défense des îles Horizon (HIDF).\nLes forces armées de la République des îles Horizon utilisent de nombreux équipements excédentaires de l'OTAN.</French>
             </Key>
+            <Key ID="STR_A3AP_setupFactionsTab_remnants">
+                <Original>Remnants.\nRemnants of the AAF forces sponsored by CSAT, who want to regain control of the islands..</Original>
+                <Russian>Remnants.\nRemnants - остатки сил AAF спонсируемые CSAT, желающие вернуть контроль над островами.</Russian>
+                <Chinesesimp>Remnants.\nPLACEHOLDER。</Chinesesimp>
+                <Korean>엑세게르메노스.\nPLACEHOLDER.</Korean>
+                <French>Remnants.\nPLACEHOLDER.</French>
```

Comment:
Might aswell just remove these translations so it uses english, unless somebody gives a translation

Reply:
Ignore the russian one, didn't realise I selected it